vmm/src/vm.rs

```diff
@@ -908,6 +929,45 @@ impl Vm {
         }
     }
 
+    fn migration_request_handle(&self, receiver: Receiver<MigrationRequest>) -> Result <()> {
```

Comment:
So this essentially is the VM migration thread, right?

Reply:
Nope, this is a child-thread in Vm to respond to 'MigrationStateGet'
request and send Vm's states to migration thread.

In fact, there will be a migration instance to be created in the future.
Just like the codes in #347, a MigrationSender on source VM side. It
sends request to component to get the states back.

In this RFC patch set, you can refer the patch 3. In 'fn boot()', I
create a thread to call 'migration_state_iter_get()' to get states.